util/labels.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) The Thanos Community Authors.
+// Licensed under the Apache License 2.0.
+
+package util
```

Comment:
I am not a big fan of `util` packages since they tend to grow in size and end up with many unrelated functions. Maybe we can call the label `extlabels` as in labels extensions or something similar to that.

Reply:
moved the package to "extlabels" !